bigchaindb/backend/localmongodb/query.py

```diff
@@ -283,12 +283,9 @@ def store_validator_set(conn, validators_update):
 
 @register_query(LocalMongoDBConnection)
 def store_election_results(conn, election):
-    height = election['height']
     return conn.run(
-        conn.collection('elections').replace_one(
-            {'height': height},
+        conn.collection('elections').insert_one(
```

Comment:
I would recommend using `replace_one` because in an event of a crash the replay won't be successful.